lib/google/longrunning/operations_pb.rb

```diff
@@ -0,0 +1,50 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
I think we've decided these pb.rb files are either in a part of googleapis-common-protos package or an independent package.